src/bindings/python/tests/test_utils/test_utils.py

```diff
@@ -58,3 +60,31 @@ def deprecated_function4():
         deprecated_function3()
     with pytest.warns(DeprecationWarning, match="deprecated_function4 is deprecated and will be removed in version 2025.4. Use another function instead"):
         deprecated_function4()
+
+
+class TestGetCmakePath(unittest.TestCase):
```

Comment:
Hm, usually we are using pytest for testing. I just want to be consistent here

Reply:
it is easier to use mocks from unittest